CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.25.0] - 2023-06-19
```

Comment:
We might have conflicts here after merging this pull request.

https://github.com/Backbase/engagements-data/pull/30/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8

cc @madz0 @amorimluzia